bundles/org.openhab.binding.jellyfin/src/main/java/org/openhab/binding/jellyfin/internal/handler/JellyfinClientHandler.java

```diff
@@ -586,17 +489,27 @@ private void handleMediaControlCommand(ChannelUID channelUID, Command command)
         }
     }
 
-    private void seekToPercentage(int percentage) throws SyncCallback.SyncCallbackError, ApiClientException {
+    private void seekToPercentage(Command command)
+            throws IllegalArgumentException, SyncCallback.SyncCallbackError, ApiClientException {
+        if (command.toFullString().equals(UnDefType.NULL.toFullString())) {
+            return;
+        }
         if (lastRunTimeTicks == 0L) {
             logger.warn("Can't seek missing RunTimeTicks info");
             return;
         }
+        int percentage = Integer.parseInt(command.toFullString());
         var seekPositionTick = Math.round(((float) lastRunTimeTicks) * ((float) percentage / 100.0));
         logger.debug("Seek to {}%: {} of {}", percentage, seekPositionTick, lastRunTimeTicks);
         seekToTick(seekPositionTick);
     }
 
-    private void seekToSecond(long second) throws SyncCallback.SyncCallbackError, ApiClientException {
+    private void seekToSecond(Command command)
+            throws IllegalArgumentException, SyncCallback.SyncCallbackError, ApiClientException {
```

Comment:
You added `IllegalArgumentException`.
What is the code that can trigger this exception ?